rpc/schemas/common/devices/WebBrowserDeviceRegistration.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "type": "object",
+  "allOf": [ { "$ref": "DeviceRegistration.json#DeviceRegistration" } ],
+  "properties": {
+    "url": { "type": "string", "format": "uri" },
+    "browser": { "type": "string" }
+  },
+  "required": [ "macAddress" ],
```

Comment:
Copy/paste error. Also shows you haven't run this through a checker manually. I think the documentation suggests to serialize, and then manually check the serialized form against this schema. There are plenty of online validators you can use.

Reply:
Not the schema. An instance of the object serialized to JSON. 🙂

I just typically run/modify a unit test, place a breakpoint, and copy the serialized value.

Reply:
Not the schema. An instance of the object serialized to JSON. 🙂

I just typically run/modify a unit test, place a breakpoint, and copy the serialized value.